providers/transip/transipProvider.go

```diff
@@ -325,6 +348,6 @@ func getTargetRecordContent(rc *models.RecordConfig) string {
 	case "SRV":
 		return fmt.Sprintf("%d %d %d %s", rc.SrvPriority, rc.SrvWeight, rc.SrvPort, rc.GetTargetField())
 	default:
-		return models.StripQuotes(rc.GetTargetCombined())
+		return models.RemoveSlashes(models.StripQuotes(rc.GetTargetCombined()))
```

Comment:
Is models.RemoveSlashes() only needed to TXT records?  If so, move that to a `case "TXT"` statement.

Reply:
I believe so... added it.